envergo/hedges/forms.py

```diff
@@ -21,9 +23,67 @@ class HedgeDataForm(forms.Form):
         """
         ),
     )
+    proximite_mare = forms.BooleanField(
+        label="Mare à moins de 200 m",
+        required=False,
+    )
+    proximite_point_eau = forms.BooleanField(
+        label="Mare ou ruisseau à moins de 10 m",
+        required=False,
+    )
+    connexion_boisement = forms.BooleanField(
+        label="Connectée à un boisement ou à une autre haie",
+        required=False,
+    )
+
+
+class HedgeToRemoveDataForm(HedgeDataBaseForm):
     sur_parcelle_pac = forms.BooleanField(
-        label="Située sur une parcelle PAC", required=False
+        label="Située sur une parcelle PAC",
+        required=False,
     )
-    proximite_mare = forms.BooleanField(
-        label="Présence d'une mare à moins de 200 m", required=False
+    vieil_arbre = forms.BooleanField(
+        label="Contient un ou plusieurs vieux arbres, fissurés ou avec cavités",
+        required=False,
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # Reorder the fields
+        self.fields = OrderedDict(
+            [
+                ("hedge_type", self.fields["hedge_type"]),
+                ("sur_parcelle_pac", self.fields["sur_parcelle_pac"]),
+                ("vieil_arbre", self.fields["vieil_arbre"]),
+                ("proximite_mare", self.fields["proximite_mare"]),
+                ("proximite_point_eau", self.fields["proximite_point_eau"]),
+                ("connexion_boisement", self.fields["connexion_boisement"]),
+            ]
+        )
+
+        # discriminate the fields id to enable two forms on the same page
```

Comment:
Est-ce que le paramètre `prefix` permettrait de répondre à cet usage ?

https://docs.djangoproject.com/en/5.1/ref/forms/api/#prefixes-for-forms